app/Routes.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
```

Comment:
Do you understand why some files are marked as deleted/created and others are recognized as renamed?

Reply:
Nope. I think this is just the scratching the limits of git magic. 

Sometimes the git magic is depending on what program you use. For example `sublime merge` would display indentation changes very badly, but online in GitLab you could see everything really good.

I mean:
```
   if condition:
       <old code with one more indent>
   else:
        some new stuff here
```